libclc/cmake/modules/AddLibclc.cmake

```diff
@@ -211,8 +211,9 @@ endfunction()
 #  * ALIASES <string> ...
 #      List of aliases
 #  * INTERNAL_LINK_DEPENDENCIES <string> ...
```

Comment:
If this comment is to be updated it should probably say `<target> ...`? What do you think?

Reply:
done